R/module_nested_tabs.R

```diff
@@ -92,22 +92,22 @@ ui_teal_module.teal_module <- function(id, modules, depth = 0L) {
   args <- c(list(id = ns("module")), modules$ui_args)
 
   ui_teal <- tagList(
-    div(
-      id = ns("validate_datanames"),
-      ui_validate_reactive_teal_data(ns("validate_datanames"))
-    ),
     shinyjs::hidden(
       tags$div(
-        id = ns("transformer_failure_info"),
+        id = ns("transform_failure_info"),
         class = "teal_validated",
         div(
           class = "teal-output-warning",
-          "One of transformers failed. Please fix and continue."
+          "One of transformators failed. Please fix and continue."
```

Comment:
What is "fix and continue"? If it's about transformators inputs then "Please check its inputs."
(this is repeating multiple times)

Reply:
Changed 3 occurences here a9a0b0e8f

I'll be in the lookout for more but haven't found any more exact matches or under `teal-output-warning`